test/cluster/lwt/lwt_common.py

```diff
@@ -0,0 +1,340 @@
+"""
+Copyright (C) 2024-present ScyllaDB
+SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
```

Comment:
Please use ordinary comments ("# ") for the boilerplate, so we can mechanically search for them if needed.